bzl/tbb.BUILD

```diff
@@ -1,67 +1,57 @@
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+
 package(default_visibility = ["//visibility:public"])
 
 exports_files(["LICENSE"])
 
+write_file(
+    name = "gen_version_file",
+    out = "version_string.ver",
+    content = [
+        "#define __TBB_VERSION_STRINGS(N) \"Empty\"",
+    ],
+)
+
+copy_file(
+    name = "copy_asm_file",
+    src = "src/tbb/intel64-masm/intel64_misc.asm",
+    out = "src/tbb/intel64-masm/intel64_misc.S",
+)
```

Comment:
This probably won't work across platforms. That's because each compiler deals with assembly differently. Currently, we don't have a good way of compiling assembly that works for all of gcc, msvc, and clang.